packages/cms/src/api/testCMS.js

```diff
@@ -0,0 +1,20 @@
+module.exports = function(app) {
+
+  const {
+    CANON_API
+  } = process.env;
+
+  app.get("/api/alive/cms", (req, res) => {
```

Comment:
This comment is very nit-picky and applies to both`testCMS.js` and `testCore.js`: I think the word "alive" makes sense in this initial case, but if we are to eventually follow up with the `TODO` here I feel like this endpoint will be more of a "status" endpoint, not just checking if it's "alive". Maybe we make these endpoints `/api/status/cms` and rename the file to `status.js`?

Reply:
Done! Good one.